### PR TITLE
feat(WaniKani): Add Extra Study details

### PR DESCRIPTION
**Pull request by theusaf**
**Describe the changes in this pull request:**
Adds details about the new Extra Study feature in WaniKani.

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

2022-03-25 13_24_57-presence-dev - Discord
2022-03-25 13_25_17-presence-dev - Discord
</details>

### CODE REVIEW

File: websites/W/WaniKani/presence.ts [hunk 1]
```diff
@@ -30,12 +30,11 @@ presence.on("UpdateData", () => {
...
 					);
 					if (buttons.length === 2) {
-						const lessons: number =
-								+buttons[0].querySelector("span").textContent,
-							reviews: number = +buttons[1].querySelector("span").textContent;
+						const lessons = +buttons[0].querySelector("span").textContent,
+							reviews = +buttons[1].querySelector("span").textContent;
 						details = "Viewing Dashboard";
 						state = `${lessons} lessons | ${reviews} reviews`;
-						smallImageText = document.querySelector<HTMLAnchorElement>(
```
Comment (EncryptedDev) [inline websites/W/WaniKani/presence.ts:38]: Why was the type removed?

**Review by EncryptedDev — DISMISSED** [review]
Across the modified changes, why were types removed from a few query selectors?

File: websites/W/WaniKani/presence.ts (hunk 1 above)
Comment (theusaf) [reply websites/W/WaniKani/presence.ts:38]: Didn't feel that it was necessary, as I'm only accessing `.textContent`. Would you prefer I add it back?

Comment (EncryptedDev) [reply websites/W/WaniKani/presence.ts:38]: Given we’re using typescript, any added types are preferable

**Review by Slowlife01 — DISMISSED** [review]
lgtm

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**